### PR TITLE
Treemap serialization

### PR DESCRIPTION
**Pull request by vorot93**

### CODE REVIEW

File: tests/treemap_serialization.rs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    assert_eq!(rb, new_rb);
+}
+
+#[test]
+fn serialization_of_empty_treemap() {
+    serialize_deserialize(vec![])
+}
+
+#[test]
+fn serialization_basic() {
```
Comment (AskAlexSharov) [inline tests/treemap_serialization.rs:23]: Probably you need to add testcase with digits > MAX_U32

**Comment by Kerollmops** [comment]
Thank you very much @vorot93 and sorry for the delay.

It seems that you added enough tests for your bitmap ser/deserialization feature, I'll ask bors to check that out and merge if that's ok to him 😄 

bors r+

**Comment by Kerollmops** [comment]
bors try

**Comment by Kerollmops** [comment]
Thank you very much for those changes, I plan to do a release soon!

**Pull request merged**